deploy/charts/cert-manager/requirements.yaml

```diff
@@ -1,7 +1,7 @@
 # requirements.yaml
 dependencies:
 - name: webhook
-  version: "v0.1.0"
+  version: "v0.1.1"
```

Comment:
Can you remove your changes to version numbers across this changeset? We automatically manage version numbers on a per-release basis, so these changes are a no-op 😄

Reply:
Sure, done (https://github.com/jetstack/cert-manager/pull/1694/commits/1e30dbfc294bdd15e00275f026351ad48c87c16a)!